driver/src/main/java/org/qbicc/driver/Driver.java

```diff
@@ -251,6 +251,26 @@ private DefinedTypeDefinition defaultFinder(ClassContext classContext, String na
         return null;
     }
 
+    private byte[] defaultResourceFinder(ClassContext classContext, String name) {
+        ByteBuffer buffer;
```

Comment:
nitpick: This declaration can be moved to the `buffer = resource.getBuffer();` line